### PR TITLE
leaky_relu and LeakyReLU: alpha->negative_slope

### PR DESCRIPTION
**Pull request by zhupengyang**
### PR types

Others

### PR changes

APIs

### Describe

- leaky_relu: alpha->negative_slope
- fix result when alpha>1
image

### CODE REVIEW

**Review by hong19860320 — DISMISSED** [review]
LGTM

**Comment by T8T9** [comment]
LGTM

File: python/paddle/nn/functional/activation.py [hunk 1]
```diff
@@ -344,6 +343,58 @@ def hsigmoid(input,
...
+            For more information, please refer to :ref:`api_guide_Name`.
+
+    Returns:
+        A Tensor with the same data type and shape as ``x`` .
+
+    Examples:
+
+        .. code-block:: python
+
+        import paddle
```
Comment (TCChenlong) [inline python/paddle/nn/functional/activation.py]: 这里缩进有问题哈 预览也有bug了 应该是少了四个空格

Comment (zhupengyang) [reply python/paddle/nn/functional/activation.py]: Done

**Review by lanxianghit — DISMISSED** [review]
LGTM

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by lanxianghit — APPROVED** [review]

**Pull request merged**